master/pkg/searcher/asha.go

```diff
@@ -243,7 +243,7 @@ func (s *asyncHalvingSearch) progress(float64) float64 {
 }
 
 func (s *asyncHalvingSearch) trialExitedEarly(
-	ctx context, requestID RequestID,
+	ctx context, requestID RequestID, exitedReason workload.ExitedReason,
```

Comment:
```suggestion
	ctx context, requestID RequestID, _ workload.ExitedReason,
```

Reply:
Eventually I'm going to be using the exitedReason here once I add in searcher-specific functionality, so thinking I should keep it like this for now rather than change it to _?